packages/docs/source/base/tokens.md

```diff
@@ -0,0 +1,151 @@
+# Design tokens
```

Comment:
Naming across tokens feels a little off to me. Other than spacing, all of our current tokens are color-related - should they share the `$color-` prefix?

Will we have non-color `$text-` tokens in the future?

Reply:
I think we should pick up our naming standards/conventions conversation - rules around naming for tokens particularly should be documented. Lets fix post 0.3.0? I think spacing/tshirt enum is another related conversation we should have at the same time